website/docs/d/aws_static_credentials.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_static_credentials data source"
+sidebar_current: "docs-vault-datasource-aws-static_credentials"
+description: |-
+  Reads static role credentials from an AWS secret backend in Vault
+---
+
+# vault\_aws\_static\_credentials
+
+Reads static role credentials from an AWS secret backend in Vault
+
+~> **Important** All data retrieved from Vault will be
+written in cleartext to state file generated by Terraform, will appear in
+the console output when Terraform runs, and may be included in plan files
+if secrets are interpolated into any resource attributes.
+Protect these artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+~> **Note**
+When using the outputs of this data source to authenticate with the [Terraform Provider for AWS](https://registry.terraform.io/providers/hashicorp/aws/latest/docs) or
+the [Terraform Provider for AWS Cloud Control](https://registry.terraform.io/providers/hashicorp/awscc/latest/docs),
+the credentials leased from Vault cannnot be renewed.
+Ensure that the lease is long enough for Terraform to complete.
+
+## Example Usage
+
+```hcl
+resource "vault_aws_secret_backend" "aws" {
+	path = "%s"
+	description = "Obtain AWS credentials."
+}
+
+resource "vault_aws_secret_backend_static_role" "role" {
+	backend = vault_aws_secret_backend.aws.path
+	name = "test"
+	username = "%s"
+	rotation_period = "3600"
+}
+
+
+# generally, these blocks would be in a different module
+data "vault_ldap_static_credentials" "creds" {
+  backend   = vault_aws_secret_backend.aws.path
+  role_name = vault_ldap_secret_backend_static_role.role.role_name
+}
+
+provider "aws" {
+  access_key = data.vault_aws_access_credentials.creds.access_key
+  secret_key = data.vault_aws_access_credentials.creds.secret_key
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Required) The path to the LDAP secret backend to
```

Comment:
The `namespace` parameter is also supported here, so we can add a block for that similar to the static role docs